Multiple mob skill update

astralAU opened the pull request:
Skill match from DP for:
Mechaspider (MVP)
Nightmare Scaraba
Bio4
Bio5

<!-- NOTE: Anything within these brackets will be hidden on the preview of the Pull Request. -->

* **Addressed Issue(s)**: 

<!--
Please specify the ChocobotRO [GitHub issue(s)](https://help.github.com/articles/autolinked-references-and-urls/#issues-and-pull-requests) this pull request amends.
If no issue exists yet, please [create one](https://github.com/Latiosu/chocobotro/issues/new) first and then link your pull request to the amendment!
-->

* **Description of Pull Request**: 

<!-- Describe how this pull request will resolve the issue(s) listed above. -->

astralAU committed:
Multiple mob skill update

Skill match from DP for:
Mechaspider (MVP)
Nightmare Scaraba
Bio4
Bio5

latiosu commented on db/re/mob_skill_db.txt:
```diff
@@ -12322,3 +12441,18 @@
+3741,MECHASPIDER@NPC_SUMMONSLAVE,attack,196,1,10000,700,10000,no,self,slavele,3,3736,,,,,,
+3741,MECHASPIDER@NPC_SUMMONSLAVE,idle,196,1,10000,700,10000,no,self,slavele,3,3736,,,,,,
+3741,MECHASPIDER@AL_HEAL,idle,28,10,10000,0,5000,yes,self,mystatuson,hiding,,,,,,,
+3741,MECHASPIDER@AL_HEAL,idle,28,10,10000,0,10000,yes,self,myhpltmaxrate,50,,,,,,,
```
Looks like this change was meant to be part of https://github.com/Latiosu/chocobotro/pull/8

latiosu replied:
Actually never mind, it makes sense for it to be here 👍

astralAU committed:
Corrections (see desc.)

Scaraba and bio4 were not correctly pulled for some reason.
Added.
Did a major pass over everything again and made a number of corrections, in particular in the yes/no column as discussed.

gruberaj commented:
I had a thought overnight that there could be some errors whereby I've put in new skills that don't exist in the source. These will need to be commented out. There won't be too many, not sure how to identify without seeing the error messages.

I've already researched as well as I can but there could be a skill level limit on DP where it only shows up to max L10. Some mobs may legitimately need to have a higher skill level of a given skill, I really don't know.

latiosu committed:
Merge remote-tracking branch 'upstream/chocobot-ro' into Mechaspider,-Scaraba-Instance,-Bio4,-Bio5-DP-Match

latiosu committed:
Adjust mob skills for One-Horn Scaraba (2161)

latiosu committed:
Adjust Two-Horn Scaraba (2162) skills

latiosu committed:
Adjust Antler Scaraba (2163) skills

latiosu committed:
Adjust Rake Scaraba (2164) skills

latiosu committed:
Adjust Queen Scaraba (2165) skills
